cs-vmware/src/main/java/io/cloudslang/content/vmware/utils/InputUtils.java

```diff
@@ -138,4 +139,10 @@ public static void checkMutuallyExclusiveInputs(final String input1, final Strin
             throw new IllegalArgumentException(exceptionMessage);
         }
     }
+
+    public static void checkOptionalMutuallyExclusiveInputs(final String input1, final String input2, final String exceptionMessage) {
+        if (isNotBlank(input1) && isNotBlank(input2)) {
```

Comment:
This is a method that checks optionally mutual exclusive (you can specify one of them or none of them; not providing value for any of the inputs is a valid use-case).